resilience4j-core/src/main/java/io/github/resilience4j/core/StringUtils.java

```diff
@@ -0,0 +1,11 @@
+package io.github.resilience4j.core;
+
+import io.github.resilience4j.core.lang.Nullable;
+
+public class StringUtils {
+
+    public static boolean isNullOrEmpty(@Nullable String string) {
```

Comment:
Can we revert the logic to `isNotEmpty`.
It's usally used like `!StringUtils.isNullOrEmpty`.
I would prefer `StringUtils.isNotEmpty`